plugins/container/go-worker/pkg/container/cri.go

```diff
@@ -378,6 +374,19 @@ func (c *criEngine) ctrToInfo(ctx context.Context, ctr *v1.ContainerStatus, podS
 			Size:             size,
 		},
 	}
+
+	if podSandboxStatus.Linux != nil && podSandboxStatus.Linux.Namespaces != nil &&
```

Comment:
I don't really know the context, but do you also need to check that podSandboxStatus is nil?